pkg/forklift-api/webhooks/validating-webhook/admitters/secret-admitter.go

```diff
@@ -76,16 +76,21 @@ func (admitter *SecretAdmitter) validateProviderSecret() *admissionv1.AdmissionR
 	if createdForProviderType, ok := admitter.secret.GetLabels()["createdForProviderType"]; ok {
 		providerType := api.ProviderType(createdForProviderType)
 
-		// The OVA secret has only name and URL fileds,
```

Comment:
so now we will be allowed to update only the secret name? what's the benefit? is the secret always provided in the request? if not we can use admitter.secret and drop the role changes + query.

Reply:
ok, so if we have the new secret (only when user specifies it?) then, once we see it we know it's a change. of course it will be better if we check the previous and say: if it's still the same URL ok, go ahead. i wonder if we need it. user wise it's better as you do it now.